acs_registry/protocols/station_query_protocol.py

```diff
@@ -4,7 +4,9 @@
 )
 from uagents import Context, Protocol
 
-from domain.station import Station
+from mongodb.get_mongo_db import get_collection
+
+station_collection = get_collection()
```

Comment:
would it be possible to move this into `mongo_db.py` and make it the only public thing
so make `get_database()` and `get_collection()` private (idk if private exists in python :) )?

Reply:
is not possible because of python